File: Selenium/Project/CRM_Activity2.java
package demosuit;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeTest;
import org.testng.annotations.Test;

public class CRM_Activity2 {
	 WebDriver driver;

	    @BeforeTest
	    public void beforeMethod() {
	        // Create a new instance of the Chrome driver
	        driver = new ChromeDriver();

	        // Open the browser
	        driver.get("https://alchemy.hguy.co/crm/");
	    }
	   
	    @Test
	    public void getHeaderImageURL() {
	        // Find the header image element using its XPath
	        WebElement headerImage = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));

	        // Get the URL of the header image
	        String imageURL = headerImage.getDomAttribute("src");
	        // Print the image URL to the console
	        System.out.println("Header Image URL: " + imageURL);
	       
	    }
	  //Tear down function
		@AfterClass
		public void tearDown() {
			// Close the browser
			driver.quit();
		}

}

File: Selenium/Project/CRMActivity2.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")

    header_image_element = driver.find_element (By.XPATH,"//img[@alt='SuiteCRM']")
    
    # Get the URL of the header image
    header_image_url = header_image_element.get_dom_attribute("src")
    
    # Print the URL of the header image
    print("Header Image URL:", header_image_url)
    
    driver.quit()